bhklab/PharmacoDI_snakemake_pipeline
# Reintegrate schema_change to main

Resolves #3 

- Update: new table schemas for Gene_Compound_Tissue_Dataset and Gene_Compound_Dataset tables
- Fix: assigning permutation_done column off of fdr_analytic instead of fdr_permutation
- Update: Made df and n nullable to work around current DB write issues * Will figure out why they are NA later and fix it to ensure the ERD is valid
- Test: added a space to see if GitHub commits show up in Slack
- Test: are Slack notifications working again?
- Update: add config.yaml
- Update: added rule to convert metaanalysis tables to .jay format for out of core joins
- Update: added conda environment export for portability
- Update: build_synonym_tables now maps dataset id correctly
- Update: build_target_tables rule now working
- Update: build_clinical_trials_table working
- Update: everything except build_metaanalysi_tables working correctly
- Update: everything working up to build_metaanalysis_tables
- Update: build_all_pset_tables working for NCI60
- Update: split gene_compound_tissue_dataset operations into their own rule * This should allow enforcement of a memory limit using datatable when joining
- Fix: all table now building; added rule to map cell_uid to cell table * All that is left is to update the schema and write to db
- Update: table schemas update
- Debug: DB write script
- Fix: chunking for batch insertion works again
- Fix: wrong column type in Dose_Response table definition causing out of bounds error
- Fix: forgot to add 1 to index in table insert statemnt * As a result, bulk insert tables were not getting their last row
- Fix: gaps in bulk inserted tables due to exclusive indexing
- Update: working on functions for table updates instead of total rewrite
